deluge/common.py

```diff
@@ -44,6 +45,13 @@
 
     os.environ['SSL_CERT_FILE'] = where()
 
+# On windows sock.if_nameindex() is only supported on 3.8+ and does not return a uuid.
+# Try ifaddr to verify network interfaces.
```

Comment:
Please remove both the comments as they don't add anything here. If anything the usage of ifaddr would be mentioned in the function.

We can look at the git commit if we need to know why it was added